### PR TITLE
Import sources from UUIDs

### PR DESCRIPTION
**Pull request by beamalsky**
An adapted `import_google_doc` script to work with the new normalized source structure. 

Source sheet it's pulling from: https://docs.google.com/spreadsheets/d/15lChqP4cKsjk8uUbUTaUA5gOUM09-t9YMIODORHw--8

Org, person, and event sheets: https://docs.google.com/spreadsheets/d/1o-K13Od1pGc7FOQ-JS8LJW9Angk3_UPZHTmOU7hD3wU/edit?ts=5b6a0b67#gid=1692418425

Notes on some errors in the source data:

- The sources in the [events sheet](https://docs.google.com/spreadsheets/d/1o-K13Od1pGc7FOQ-JS8LJW9Angk3_UPZHTmOU7hD3wU/edit?ts=5b6a0b67#gid=764217023) are still strings, not UUIDs.
- For the orgs sheet, it looks like most dates are in `month/day/year` format but some are `day/month/year`. The event sheet looks like it may be all `day/month/year`. These are detailed in the error logs. We're not currently importing them as such, but should figure out if this is part of the main data as well.

### CODE REVIEW

**Comment by beamalsky** [comment]
@evz I've got the dates from the sources sheet parsing correctly now, could I get your eyes on this to make sure I didn't upset anything else with my change to `parse_date`?

**Comment by beamalsky** [comment]
@evz I ran `import_google_doc` after manually importing the OSM data for Mexico and it had no unexpected trouble with OSM IDs, fyi

**Comment by tlongers** [comment]
@beamalsky

> For the orgs sheet, it looks like most dates are in month/day/year format but some are day/month/year. The event sheet looks like it may be all day/month/year. These are detailed in the error logs. We're not currently importing them as such, but should figure out if this is part of the main data as well.

We also specify  `mm`,`dd` and `yyyy` in the three columns that follow any date column. These could either be used as the primary source of data for a date, or a quick integrity check on the date parsed out of the date column itself.

File: sfm_pc/management/commands/import_google_doc.py [hunk 1]
```diff
@@ -746,8 +782,12 @@ def make_relation(self,
...
 
         try:
-            value = data[value_position]
-            value = value.strip()
+            if not date:
+                value = data[value_position]
+                value = value.strip()
+            else:
+                date_parts = [data[value_position + 3], data[value_position + 1], data[value_position + 2]]
+                value = '-'.join(filter(None, date_parts))
```
Comment (evz) [inline sfm_pc/management/commands/import_google_doc.py]: This is a very nice solution. I take it these positions are consistent?

Comment (beamalsky) [reply sfm_pc/management/commands/import_google_doc.py]: they are!

**Comment by beamalsky** [comment]
@evz I fixed the loop logic in `make_emplacement` and `make_area`, I think this is good to merge.

**Comment by evz** [comment]
I'm going to go ahead and merge this since I managed to isolate the things I needed to deploy to the production server elsewhere.

**Pull request merged**